types/Overview.ts

```diff
@@ -43,6 +45,7 @@ export interface OverviewFindingItem {
   ruleId: string;
   ruleName: string;
   ruleSeverity: string;
+  isThreatIntelOnlyFinding: boolean;
```

Comment:
is this a new addition, why?

Reply:
The interfaces are duplicated in two places. It was updated in the other place but not here. Added it here and also removed (or will remove it as part of next PR) the other instance so that we have them defined only in one place